scraper/README.md

```diff
@@ -100,5 +100,16 @@ The corrected data in the new file should be stored in the same format as the ge
 The scraper also inserts the scraped data into the database:
 
 ```shell
-npm run insert
```

Comment:
A `npm run insert` which does both insert:plants and insert:relations would be handy, too (and documented). Ideally the `npm run insert` in the scraper should be also added to Makefile.
ci/Jenkinsfile line 190 must be adapted to also copy the Companions.csv` and `Antigonist.csv` files.